crates/re_viewer_context/src/gpu_bridge/tensor_to_gpu.rs

```diff
@@ -145,13 +154,20 @@ pub fn color_tensor_to_gpu(
     // Assume that the texture is not pre-multiplied if it has an alpha channel.
     let multiply_rgb_with_alpha = depth == 4;
 
+    let gamma = if encoding == Some(TextureEncoding::Nv12) {
+        2.2
+    } else {
+        1.0
+    };
```

Comment:
We don't actually want to set the gamma value here but instead flip the `decode_srgb` bool - the expected color space after decoding is srgb-gamma space and we want to go to srgb-linear from there and that's what the flag does. This results in a few very slight color differences, also we apply gamma right now _after_ filtering which further alters the result (today, we use gamma as an inspection option in the Tensor space view only)


There's just the problem that we don't actually go through decode srgb path yet in the shader and doing so _optionally_ doesn't really make sense for NV12. So in any case we want to have a `decode_nv12_to_linearsrgb` in the shader. I think to solve this we should actually join the new `encoding` and `decode_rgb` together to a single `shader_decode` option that does the right thing then. But I'm ok with taking this as a follow-up :)

Reply:
Will check in the evening

Reply:
Hmm, didn't even realize the bool was there before. I think the shader actually does go through the decode srgb path if you wrap the textureLoad in `decode_color`, which all texture loads are already (except nv12). I reverted the manual gamma and went the decode_srgb path, which also uncovered a few clamping related bugs in the decoder, fixed those aswell.

Reply:
sounds good!